File: Python/Construct the Rectangle.py
class Solution(object):
    def constructRectangle(self, area):
        """
        :type area: int
        :rtype: List[int]
        """
        for l in xrange(int(area ** 0.5), area +1):
            if area % l == 0:
                return [max(l , area / l), min(l , area/ l)]
        

File: cpp/Construct the Rectangle.cpp
class Solution {
public:
    vector<int> constructRectangle(int area) {
        vector<int> result;
        for (int i = (int) sqrt(area); i < area + 1; ++i) {
            if (area % i == 0) {
                result.push_back(max(area / i, i));
                result.push_back(min(area / i, i));
                break;
            }
        }
        return result;
    }
};
